plugin/plugin_test.go

```diff
@@ -0,0 +1,226 @@
+// Copyright 2020 the Drone Authors. All rights reserved.
+// Use of this source code is governed by the Blue Oak Model License
+// that can be found in the LICENSE file.
+
+package plugin
+
+import (
+	"context"
+	"fmt"
+	"testing"
+)
+
+func TestUnitTcClean(t *testing.T) {
+	args := GetArgsForFunctionalTesting(
+		getDefaultPluginDriverPath(),
+		"clean",
+		getDefaultPluginLocations(),
+		getDefaultPluginCommandLineArgs(),
+		getDefaultPluginUrl(),
+		getDefaultPluginUser(),
+		getDefaultPluginPassword(),
+	)
+
+	fp, err := Exec(context.TODO(), args)
+	if err != nil {
+		fmt.Println("Error in Exec: " + err.Error())
+		t.Fail()
```

Comment:
can you replace t.Fail() statements with `require` and add failure messages as well

Reply:
done